lib/prepare.js

```diff
@@ -39,6 +40,7 @@ async function createDirectories() {
 
     directories.in   = path.resolve(directories.base, IN_DIRECTORY);
     directories.out  = path.resolve(directories.base, OUT_DIRECTORY);
+    directories.post = path.resolve(directories.out, POST_PROC_DIRECTORY);
```

Comment:
Can we rename it to `postprocessing`? 

In subsequent code I got confused for a second thinking this was HTTP(S) related to the `POST` verb...

Reply:
That would be too long IMO (paths are already long, making it hard to read in logs). I agree `post` is not very clear - any other suggestion that makes it more clear but is still short?

Reply:
`P_PROCESSING_DIRECTORY`
`POST_WORK_DIRECTORY`
`PROCESSING_DIRECTORY`
`SCRATCH_DIRECTORY`

and then make the actual directory name `postprocessing`?

Reply:
I was referring to the directory name itself (currently `post`). Ok with renaming the `directories.*` field, and the constant is already specific `POST_PROC_DIRECTORY` (I think).

Reply:
`directories.postProc`?  it is not used that often, not the worst idea to use the full name
and +1 for still calling the actual directory `postprocessing`

Reply:
Changed code to

```
    directories.postprocessing = path.resolve(directories.out, POST_PROCESSING_DIRECTORY);
```

but kept the dir name as `post`.

Reply:
@jdelbick we cross posted :-)

Here is a real world example of the path:

```
2020-09-30T23:17:35.251212700Z [rendition 0] WORKER_TEST_MODE: moving /nodejsAction/08fMiXQm/work/kfq0fcje/out/post/rendition0.png to /nodejsAction/08fMiXQm/work/kfq0fcje/out/rendition0.png
```

It's already pretty long (and `kfq0fcje` will be activation id in production so even longer).

First and foremost I introduced the directory to avoid file name conflicts, i.e. to be able to have the same file in `out/file.png` as in `out/post/file.png`. This simplified a few things, also that we could just use Rendition() constructor without a `name` param but just change the `directory`.

It's not a directory that humans browse into since it's transient already. The logs show it at the beginning:

```
2020-09-30T23:17:35.082486900Z work directory        : /nodejsAction/08fMiXQm/work/kfq0fcje
2020-09-30T23:17:35.082712400Z - source directory    : /nodejsAction/08fMiXQm/work/kfq0fcje/in
2020-09-30T23:17:35.082988500Z - renditions directory: /nodejsAction/08fMiXQm/work/kfq0fcje/out
2020-09-30T23:17:35.083213500Z - post-processing dir : /nodejsAction/08fMiXQm/work/kfq0fcje/out/post
```

and I think it will be clear for us what `out/post` is since it's always the same. In this situation I feel we win more by having a bit shorter paths and thus slightly less parsing and scrolling when looking for _other_ interesting elements in the logs.